app/forms/validators.py

```diff
@@ -405,13 +405,21 @@ def format_playback_value(value, currency=None):
     return format_number(value)
 
 
+def format_message_with_title(error_message, question_title):
+    return error_message % dict(question_title=question_title)
```

Comment:
Given that the `safe_content` call is needed in multiple places, I think it would make sense to do it here instead.